Samples/SDKFeatures/SDKFeatures/SDKFeatures-Bridging-Header.h

```diff
@@ -0,0 +1,10 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
+
+#import "MHVLib.h"
+#import "MHVSodaConnectionProtocol.h"
+#import "MHVConnectionFactory.h"
+#import "MHVFeaturesConfiguration.h"
+#import "MHVConnectionFactoryProtocol.h"
```

Comment:
I cleaned up the headers in mu most recent pull request, if you wait until my PR is merged, you can just import MHVLib.h